src/test/java/core/basesyntax/dao/DataToCsvFileImplTest.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class DataToCsvFileImplTest {
```

Comment:
also lets add test to write data to null file name. Also we can att test to write empty or null data